app/templates/components/addon-tabs.hbs

```diff
@@ -5,21 +5,21 @@
       @tabId={{0}}
       @onClick={{action (mut this.currentTab) 0}}
     >
-      SASS
+      Manage State
```

Comment:
Maybe it's just me but when I read this title I never expected to find information about ember-concurrency. I immediately thought of things like Ember Data / Services / Redux / etc.

Maybe something like `Async Code` or `Async State` might be better here?